packages/web/src/common/store/upload/sagas.ts

```diff
@@ -429,7 +430,7 @@ export function* handleUploads({
   // Channel to listen for responses
   const responseChannel = yield* call(
     channel<UploadTrackResponse>,
-    buffers.expanding(10)
+    buffers.expanding(200)
```

Comment:
Tbh not sure this makes any difference - the `expanding` means it grows anyway

Reply:
i'm concerned that under the hood in react native something weird happens with this, but i agree low probability this works

Reply:
just don't understand why this complexity is even necessary in this codebase in general